src/NuGet.Indexing/NuGetAcronymExpansionProvider.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace NuGet.Indexing
+{
+    public class NuGetAcronymExpansionProvider : IAcronymExpansionProvider
+    {
+        public static readonly IAcronymExpansionProvider Instance = new NuGetAcronymExpansionProvider();
+
+        private static readonly Dictionary<string, string[]> Acronyms = new Dictionary<string, string[]>(StringComparer.OrdinalIgnoreCase);
+
+        static NuGetAcronymExpansionProvider()
+        {
+            Acronyms.Add("xaml",  new[] { "xaml" });
+            Acronyms.Add("mvc",   new[] { "mvc" });
+            Acronyms.Add("wpf",   new[] { "wpf" });
+            Acronyms.Add("ef",    new[] { "ef", "entity", "framework", "entity framework" });
```

Comment:
Do we need both entity framework and entity framework? Wouldn't the other tokenizer break it down itself later?